### DISCUSSION TITLE
Qual melhor cartão multimoeda para viagens

### DISCUSSION THREAD

**Discussion by pedrohyvo** [DyegoCosta/trabalhando-remoto#199] [General]
Gostaria de saber qual o cartão multimoeda que vocês recomendam utilizar para viagens internacionais na função de débito mesmo e para eventuais saques (se necessário). O que vocês utilizam e qual a experiência de vocês?

Atualmente recebo meu salário pela Payoneer. Caso alguém utilize o cartão deles ou já usou, me conta um pouco também :)

**Comment by marcosvieiraftw** [comment]
Na minha experiência o cartão Payoneer foi sempre o melhor, até já emprestei para amigos em viagens para miami.

**Comment by pedrohyvo** [comment]
@marcosvieiraftw você sabe como funciona a questão de sacar dinheiro fora do país e se existe uma taxa de manutenção? Eu li rapidamente sobre isso que eles cobram agora uma taxa, daí eu queria confirmar isso com alguém que já utiliza.

**Comment by marcosvieiraftw** [comment]
Parece que tem uma taxa fixa para sacar no ATM - se não me engano algo em torno de 4˜5 dólares, então não vale a pena ficar sacando picado, tem que sacar o máximo que conseguir. Fora isso nenhuma outra taxa.. Mas vale dar uma pesquisada mais a fundo nas letras miúdas lá, já tem tempo que usei então pode ter mudado.

**Comment by megurost** [comment]
Eu uso o da Payoneer e nunca tive problemas pra compra ou saque, mas também não conheço outros. Em viagens quando for sacar o ideal é sacar valores mais altos, porque senão a taxa de saque da Payoneer + taxa do ATM acabam superando a diferença que a gente pagaria entre a cotação comercial e a turismo.

Geralmente qualquer ATM de banco grande funciona, acho que um ou outro não funcionou comigo, mas aí é só achar o de outro banco. Já fiz saques no Brasil, Uruguai, Europa e Japão e deu tudo certo. Compras com o cartão funcionam em todo lugar também, inclusive costumo usar o cartão pra pagar passagens, compras do AliExpress, da Amazon e qualquer outro lugar que aceite cobrar direto na moeda do cartão.

Dá uma olhada [aqui](https://www.payoneer.com/about/fees/) que tem as taxas do cartão (tem que clicar pra expandir a parte que diz "OPTIONAL: Use the Payoneer Prepaid MasterCard®"). As taxas dos ATM variam de banco pra banco, mas costumam ser um valor fixo por saque.

Eles dizem que tem uma taxa anual de manutenção de $29.95 / €24.95 / £24.95, mas até hoje não me cobraram. Uma vez perguntei no chat e disseram que era cobrado quando tivesse saldo na conta, mas desde então já mantive valores lá várias vezes e não houve cobrança. Não sei também se essa anuidade é só caso você não use igual uns cartões daqui... seria interessante saber de outra pessoa que usa.
